lightwood/encoders/categorical/multihot.py

```diff
@@ -0,0 +1,25 @@
+import torch
+import numpy as np
+from lightwood.encoders import BaseEncoder
+from sklearn.preprocessing import MultiLabelBinarizer
+
+
+class MultiHotEncoder(BaseEncoder):
+    def __init__(self, is_target=False):
+        super().__init__(is_target)
+        self.max_words_per_sent = None
+        self.binarizer = MultiLabelBinarizer()
+
+    def prepare_encoder(self, column_data, max_dimensions=100):
+        self.binarizer.fit(column_data)
+        self._prepared = True
+
+    def encode(self, column_data):
+        data_array = self.binarizer.transform(column_data)
+        return self._pytorch_wrapper(data_array)
+
+    def decode(self, vectors):
+        # It these are logits output by the neural network, we need to treshold them to binary vectors
+        vectors = np.where(vectors > 0, 1, 0)
```

Comment:
Just one question: does it not make more sense to set the thershold at e.g. 0.5, since the networks use ReLU up until the last layer => no negative numbers until then. But maybe the intuition I have here is just bad, I think it would be worth running the experiment in the future, but unless you can do it quickly and notice some improvements I'm 100% fine with keeping it this way for now.

Reply:
I think our networks use SeLU. Either way the output vectors of our current nn mixer have negative numbers. Changing the threshold to 0.5 seems to provide no improvenment